clients/nodejs/modules/JsonRpcServer.js

```diff
@@ -857,7 +857,7 @@ class JsonRpcServer {
                 }
                 try {
                     const methodRes = await this._methods.get(msg.method).apply(null, msg.params instanceof Array ? msg.params : [msg.params]);
-                    if (msg.id) {
+                    if (typeof msg.id === 'string' || Number.isInteger(msg.id)) {
```

Comment:
Still does not allow the `null` value, I think?

Reply:
```suggestion
                    if (typeof msg.id === 'string' || Number.isInteger(msg.id) || msg.id === null) {
```

Reply:
I just did what @styppo suggested ^^

>So I think it makes sense to […] reject everything else, including NULL and non-integer numbers.


Isn't `null` is too similar to `undefined`/no ID at all from a JS perspective?

Reply:
Ah, right, did not see that. Ok, cool!